### PR TITLE
Update YAMLs and Chart for Filesystem Trim

### PR DESCRIPTION
**Pull request by shuo-wu**
#836

### CODE REVIEW

**Comment by shuo-wu** [comment]
One minor question: The YAML in this repo does not contain the ConfigMap `default-settings`. Should we add it?

File: examples/storageclass.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ parameters:
 #  backingImageDataSourceType: "download"
 #  backingImageDataSourceParameters: '{"url": "https://backing-image-example.s3-region.amazonaws.com/test-backing-image"}'
 #  backingImageChecksum: "SHA512 checksum of the backing image"
+#  unmapMarkSnapChainRemoved:
```
Comment (innobead) [inline examples/storageclass.yaml:19]: Give the possible values of the value.

**Comment by innobead** [comment]
> One minor question: The YAML in this repo does not contain the ConfigMap `default-settings`. Should we add it?

@shuo-wu We have that in default-setting.yaml and the generated deploy/longhorn.yaml, or what you mean is different?

**Review by innobead — APPROVED** [review]
LGTM

**Comment by shuo-wu** [comment]
> We have that in default-setting.yaml and the generated deploy/longhorn.yaml, or what you mean is different?
@innobead there is no content in ConfigMap `deploy/longhorn.yaml/longhorn-default-setting`:
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: longhorn-default-setting
  namespace: longhorn-system
  labels:
    app.kubernetes.io/name: longhorn
    app.kubernetes.io/instance: longhorn
    app.kubernetes.io/version: v1.4.0-dev
data:
  default-setting.yaml: |-
```

In the old manager file, we will list all options and comment them.

**Comment by innobead** [comment]
> > We have that in default-setting.yaml and the generated deploy/longhorn.yaml, or what you mean is different?
> > @innobead there is no content in ConfigMap `deploy/longhorn.yaml/longhorn-default-setting`:
> 
> ```
> apiVersion: v1
> kind: ConfigMap
> metadata:
>   name: longhorn-default-setting
>   namespace: longhorn-system
>   labels:
>     app.kubernetes.io/name: longhorn
>     app.kubernetes.io/instance: longhorn
>     app.kubernetes.io/version: v1.4.0-dev
> data:
>   default-setting.yaml: |-
> ```
> 
> In the old manager file, we will list all options and comment them.

I see, then we don't need this anymore, because users can check the values.yaml to know what settings are provided.

**Review by innobead — APPROVED** [review]

**Pull request merged**